packages/core/src/anchor/memory/in-memory-anchor-service.ts

```diff
@@ -1,43 +1,32 @@
 import { CID } from 'multiformats/cid'
 import { Observable, Subject, concat, of } from 'rxjs'
 import { filter } from 'rxjs/operators'
-import {
+import { TestUtils } from '@ceramicnetwork/common'
```

Comment:
this is an awesome cleanup.  Nice to see the InMemoryAnchorService finally reflect the way the CAS actually works these days again